piet/src/color.rs

```diff
@@ -171,6 +171,28 @@ impl Color {
         }
     }
 
+    /// Convert a color value to four 8-bit rgba values.
+    pub fn as_rgba_u8(&self) -> (u8, u8, u8, u8) {
+        let rgba = self.as_rgba_u32();
+        (
+            (rgba >> 24 & 255) as u8,
+            ((rgba >> 16) & 255) as u8,
+            ((rgba >> 8) & 255) as u8,
+            (rgba & 255) as u8,
+        )
+    }
+
+    /// Convert a color value to four f64 values, each in the range 0.0 to 1.0.
+    pub fn as_rgba_f64(rgba: &Color) -> (f64, f64, f64, f64) {
```

Comment:
our naming convention here seems to be that we assume `f64`; so this can just be `as_rgba`.
(I'll make this change, since I'd like to get this in before doing a new piet release)

```suggestion
    pub fn as_rgba(rgba: &Color) -> (f64, f64, f64, f64) {
```